ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-ocp4.14-lp-interop.yaml

```diff
@@ -41,6 +41,23 @@ tests:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "OSSM"},
```

Comment:
Shouldn't this be our project? If the cluster fails to provision, we probably just need to re-run the job. Same with the `install-operators`, `gather-*`, and `openshift-*` steps. 

This might be a bigger conversation with Paige, Aditya, and @btjd though.

Reply:
Yes I was bit confused about where those tickets will be filled. Some infra issues we need to investigate.  But not sure about where to file the operator installation issues, aws issues

Reply:
+1 to create OSSM issue only if the problem is related to OSSM. Operator installation, servicemesh-smcp-install, servicemesh-kiali-cypress-tests-execute, servicemesh-kiali-int-tests-execute, servicemesh-maistra-test-tool-execute steps issues should go to OSSM jira project. Other steps are not related to OSSM product. Otherwise LGTM